<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1362</width>
    <height>834</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Test visualization</string>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="locale">
    <locale language="English" country="UnitedStates"/>
   </property>
   <widget class="QGraphicsView" name="graphicsView_original">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>80</y>
      <width>512</width>
      <height>512</height>
     </rect>
    </property>
   </widget>
   <widget class="QGraphicsView" name="graphicsView_response">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>80</y>
      <width>512</width>
      <height>512</height>
     </rect>
    </property>
    <property name="sizeAdjustPolicy">
     <enum>QAbstractScrollArea::AdjustToContents</enum>
    </property>
   </widget>
   <widget class="QLabel" name="title_original">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>20</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Century Gothic</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Original image</string>
    </property>
   </widget>
   <widget class="QLabel" name="title_response">
    <property name="geometry">
     <rect>
      <x>970</x>
      <y>20</y>
      <width>231</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Century Gothic</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Response of layer N</string>
    </property>
   </widget>
   <widget class="QLabel" name="title_dataset">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>20</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Century Gothic</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Parameters</string>
    </property>
   </widget>
   <widget class="QTreeWidget" name="tree_network">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>80</y>
      <width>211</width>
      <height>401</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="sizeAdjustPolicy">
     <enum>QAbstractScrollArea::AdjustToContents</enum>
    </property>
    <property name="autoExpandDelay">
     <number>-1</number>
    </property>
    <property name="indentation">
     <number>0</number>
    </property>
    <column>
     <property name="text">
      <string notr="true">1</string>
     </property>
    </column>
   </widget>
   <widget class="QTreeWidget" name="tree_data">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>510</y>
      <width>211</width>
      <height>281</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string notr="true">Dataset</string>
     </property>
    </column>
   </widget>
   <widget class="QLabel" name="model_name_title">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>630</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Century Gothic</family>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Model name(s):</string>
    </property>
   </widget>
   <widget class="QLabel" name="model_name">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>630</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Century Gothic</family>
      <pointsize>14</pointsize>
      <italic>true</italic>
     </font>
    </property>
    <property name="text">
     <string>xxxxxxxxxxxxxxxxx</string>
    </property>
   </widget>
   <widget class="QLabel" name="direction">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>680</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Century Gothic</family>
      <pointsize>14</pointsize>
      <italic>true</italic>
     </font>
    </property>
    <property name="text">
     <string>From B to A</string>
    </property>
   </widget>
   <widget class="QLabel" name="direction_title">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>680</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Century Gothic</family>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Direction</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_change_direction">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>670</y>
      <width>151</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Change direction</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="load_dataset">
   <property name="text">
    <string>Load dataset...</string>
   </property>
  </action>
  <action name="load_model">
   <property name="text">
    <string>Load model...</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
